### ISSUE TITLE
Whitespace Handling

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/cicero-ui#43]
Loading text with whitespace does not always round-trip in the Clause Editor. For instance the bit of sample text from the volume discount:
```
a) Settlement Amount. Our agent will pay you according to your payment plan, as described below, in US dollars for the face amount of Charges submitted from your Establishments less all applicable deductions, rejections, and withholdings, which include: 
    (i) the Discount, 
    (ii) any amounts you owe us or our Affiliates, 
    (iii) any amounts for which we have Chargebacks, and (iv) any Credits you submit. Our agent will subtract the full amount of all applicable deductions, rejections, and withholdings, from this payment to you (or debit your Bank Account), but if it cannot, then you must pay it promptly upon demand.
```

Becomes the following in the markdown window:
```
a) Settlement Amount. Our agent will pay you according to your payment plan, as described below, in US dollars for the face amount of Charges submitted from your Establishments less all applicable deductions, rejections, and withholdings, which include: 
(i) the Discount, 
(ii) any amounts you owe us or our Affiliates, 
(iii) any amounts for which we have Chargebacks, and (iv) any Credits you submit. Our agent will subtract the full amount of all applicable deductions, rejections, and withholdings, from this payment to you (or debit your Bank Account), but if it cannot, then you must pay it promptly upon demand. 
```

Which leads to parsing issues.

**Comment by jeromesimeon** [comment]
This can be seen _in vivo_ on the following branch: https://github.com/accordproject/cicero-ui/tree/js-volume-discount-test

**Comment by irmerk** [comment]
@dselman @jeromesimeon curious if this issue will be closed when `cicero v0.20` is released?

**Comment by jeromesimeon** [comment]
> @dselman @jeromesimeon curious if this issue will be closed when `cicero v0.20` is released?

It should be... help on testing would be welcome!

**Comment by irmerk** [comment]
Closed with [this release](https://github.com/accordproject/cicero-ui/releases/tag/v0.2.0)
